nixjs-rt/src/builtins.ts

```diff
@@ -354,12 +355,18 @@ export function getBuiltins() {
         throw builtinBasicTypeMismatchError("import", pathStrict, expected);
       }
 
-      const pathValue = pathStrict.toJs();
+      let pathValue = "";
+      if (pathStrict instanceof NixString) {
+        // TODO: Get relative paths working
```

Comment:
Relative paths aren't supported in import when using strings. Here's an example:
```bash
% nix eval --impure --expr 'let flake = builtins.import "./flake.nix"; in flake.description'            
error:
       … while calling the 'import' builtin

         at «string»:1:13:

            1| let flake = builtins.import "./flake.nix"; in flake.description
             |             ^

       … while realising the context of a path

         at «none»:0: (source not available)

       error: string './flake.nix' doesn't represent an absolute path
```